test/unit/effects.js

```diff
@@ -1780,7 +1781,8 @@ test("Animation callbacks (#11797)", 15, function() {
 			ok( true, "empty: start" );
 		},
 		progress: function( anim, percent ) {
-			equal( percent, 0, "empty: progress 0" );
+			equal( percent, prog, "empty: progress " + prog );
```

Comment:
I change something here, because in jQuery if there is no property to animate (here it's empty with the `{}`) `.stop( true )` is called. So, progress is now called twice.